docs/excel/excel-add-ins-worksheets.md

```diff
@@ -326,6 +326,52 @@ Excel.run(function (context) {
 > - For examples that show how to get ranges from a `Table` object, see [Work with tables using the Excel JavaScript API](excel-add-ins-tables.md).
 > - For examples that show how to search a large range for multiple sub-ranges based on cell characteristics, see [Work with multiple ranges simultaneously in Excel add-ins](excel-add-ins-multiple-ranges.md).
 
+## Filter data
+
+> [!NOTE]
+> `AutoFilter` is currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+An [AutoFilter]((/javascript/api/excel/excel.autofilter) applies data filters across a range within the worksheet. This is created with `Worksheet.autoFilter.apply`, which has the following parameters:
```

Comment:
Remove the extra opening parenthesis in `[AutoFilter]((/javascript/api/excel/excel.autofilter)` that's making the link not render correctly.